src/Inspector/Overlay.tsx

```diff
@@ -143,7 +143,7 @@ class OverlayTip {
       color: '#d7d7d7',
     })
 
-    this.tip.style.zIndex = '10000000'
+    this.tip.style.zIndex = '10000001'
```

Comment:
is this really necessary?

this hardcode mirror from https://github.com/facebook/react/blob/v16.13.1/packages/react-devtools-shared/src/backend/views/Highlighter/Overlay.js#L44

Reply:
I didn't want to adjust this too, but without this, when keys are pressed, the overlay is behind some blue overlay, you can try cloning from my branch

Reply:
please list some screenshots or a online page?

Reply:
![Screenshot 2021-11-02 at 2 51 40 PM](https://user-images.githubusercontent.com/653805/139799649-0fe54821-e058-406d-9afb-50508fe63a21.png)